### PR TITLE
Update azure.md - no redis cluster protocol support for a successful …

### PR DESCRIPTION
**Pull request by dwaiba**
…active active setup.

Updated with:
Presently Terraform Enterprise active active does not support the redis cluster protocol and hence no clustering should be enabled for a successful active active setup.

## Labels

- [ ] inaccuracy
- [x] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout
- [ ] api - requires an update to the [changelog](https://www.terraform.io/docs/cloud/api/changelog.html)

### CODE REVIEW

**Comment by laurapacilio** [comment]
@eastebry Could you review this, or let me know who is the right person? Thank you!

**Review by practicalint — APPROVED** [review]
Yes, I think this is a helpful and valid edit. Thanks. Approved

File: content/source/docs/enterprise/before-installing/reference-architecture/azure.md [hunk 1]
```diff
@@ -372,6 +372,8 @@ Enterprise-grade security is inherently covered in the Azure Cache for Redis imp
 
 Terraform Enterprise supports Redis versions 4.0 and 5.0, but 5.0 is recommended unless there is strong reason to deviate. Azure Cache for Redis supports 5.0 and 6.0 as a preview mode, so still remain with 5.0 until 6.0 has been certified everywhere and do explicitly specify the version. The minimum TLS version can also be configured and defaults to 1.0 - you should explicitly set it to 1.2 for latest.
 
+Presently Terraform Enterprise active active does not support the redis cluster protocol and hence no clustering should be enabled for a successful active active setup.
```
Comment (laurapacilio) [inline content/source/docs/enterprise/before-installing/reference-architecture/azure.md:375]: Just revising for active voice and to make sure we're reference Active/Active the same way throughout the page. 

```suggestion
Terraform Enterprise _Active/Active_ does not currently support the Redis cluster protocol, so you should not enable clustering for a successful _Active/Active_ setup.
```

**Review by laurapacilio — CHANGES_REQUESTED** [review]
Thank you for submitting this! I left a quick suggestion for you to review. Once you take a look, we can get this merged in.

**Review by laurapacilio — APPROVED** [review]

**Pull request merged**